types/imagemin-gifsicle/index.d.ts

```diff
@@ -1,16 +1,33 @@
-// Type definitions for imagemin-gifsicle 5.2
+// Type definitions for imagemin-gifsicle 7.0
 // Project: https://github.com/imagemin/imagemin-gifsicle#readme
 // Definitions by: Romain Faust <https://github.com/romain-faust>
+//                 Piotr Błażejewicz <https://github.com/peterblazejewicz>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 import { Plugin } from 'imagemin';
 
+/**
+ * Imagemin plugin for {@link https://www.lcdf.org/gifsicle/|Gifsicle}
```

Comment:
Is there really a `|` in the URL?

Reply:
oh, no, it's just (probably) less used `@link` syntax:
![image](https://user-images.githubusercontent.com/14539/94851937-6f281480-0429-11eb-97c4-2c84b997939a.png)